eventdemo/resources/projects.xml

```diff
@@ -564,7 +564,7 @@ If you want to modify the sample application source code or create your own appl
         <sourceArchive>android/event_demo.tar.gz</sourceArchive>
         <projectFolder>event_demo/EventDemo</projectFolder>
         <sdkLibDir>event_demo/EventDemo/app/libs</sdkLibDir>
-        <destBinaryFile>event_demo/EventDemo/app/build/outputs/apk/app-debug.apk</destBinaryFile>
+        <destBinaryFile>event_demo/EventDemo/app/build/outputs/apk/events.apk</destBinaryFile>
```

Comment:
Is this a special case? I expect this should be 'event.apk' as for the rest of applications.